pkg/kms/service.go

```diff
@@ -67,3 +74,29 @@ func (s *service) CreateKey(keystoreID string, kt kms.KeyType) (string, error) {
 
 	return keyID, nil
 }
+
+// Sign signs a message using key identified by keyID.
+func (s *service) Sign(keystoreID, keyID string, msg []byte) ([]byte, error) {
+	k, err := s.keystore.Get(keystoreID)
+	if err != nil {
+		return nil, fmt.Errorf(getKeystoreErr, err)
+	}
+
+	if len(k.KeyIDs) == 0 {
+		return nil, errors.New(noKeysErr)
+	}
+
+	for _, id := range k.KeyIDs {
+		if id == keyID {
+			break
+		}
+		return nil, errors.New(invalidKeyIDErr)
```

Comment:
I'm surprised the linter doesn't complain about the line spacing here... is it set up on this project?

Reply:
will double check if it works